.env.development.EXAMPLE

```diff
@@ -1,3 +0,0 @@
-GATSBY_ALGOLIA_APPLICATION_ID=""
-GATSBY_ALGOLIA_SEARCH_KEY=""
-GATSBY_ALGOLIA_INDEX="businesses"
```

Comment:
@molebox looks like this may have been accidentally removed!

Reply:
oh sorry, i just renamed it. Can add back